__test__/(not-authenticated)/sign_up.test.jsx

```diff
@@ -93,12 +168,53 @@ describe('Sign Up', () => {
   });
 
   it('renders a Sign In link', () => {
-    render(<SignUp />);
+    render(
+      <AuthLayoutProvider>
+        <SignUp />
+      </AuthLayoutProvider>
+    );
 
     const link = screen.getByRole('link', {
       name: 'Log in',
     });
 
     expect(link).toBeInTheDocument();
   });
+
+  const findEmailInput = () => screen.getByRole('textbox', { name: 'Email' });
+
+  const findFullNameInput = () =>
+    screen.getByRole('textbox', { name: 'Full Name' });
```

Comment:
WDYT of send the value and not hardcode inside the function

Reply:
I'd rather have a little duplication than do the wrong abstraction, specially when it's so little. I like to follow WET ([Write Everything Twice](https://dev.to/wuz/stop-trying-to-be-so-dry-instead-write-everything-twice-wet-5g33))